view/adminhtml/web/js/view/certificates-grid.js

```diff
@@ -13,17 +13,35 @@
  */
 define([
     'jquery',
-    'Magento_Ui/js/form/element/abstract'
+    'Magento_Ui/js/form/element/abstract',
+    'mage/translate',
+    'Magento_Ui/js/modal/confirm'
 ], function (
     $,
-    Abstract
+    Abstract,
+    $t,
+    confirm
 ) {
     'use strict';
 
     return Abstract.extend({
         defaults: {
             template: 'ClassyLlama_AvaTax/view/certificates-grid',
             shouldShowWarning: false
+        },
+
+        handleDelete: function(certificate) {
```

Comment:
No need to pass this much data around, we have access to the certificates list in this method. Instead, you can pass an identifier, such as the index